signac/core/json.py

```diff
@@ -2,6 +2,9 @@
 # All rights reserved.
 # This software is licensed under the BSD 3-Clause License.
 import logging
+import json
```

Comment:
See if we can use this suggestion and remove the functions defined below.
```suggestion
from json import load, loads
```

Reply:
I guess we can not use `from json import load, loads` I am getting the following error for the same  
![Screenshot from 2020-02-12 02-36-45](https://user-images.githubusercontent.com/26172160/74279392-d2e9ba00-4d40-11ea-9d70-183e0c35b0ba.png)


Also while testing manually I encountered the following issue when I call `json.loads('["foo", {"bar":["baz", null, 1.0, 2]}]')`(In the previous code where we have load and loads function defined). It is getting stuck in recursion. Can you suggest if there is some flaw in my approach?
![Screenshot from 2020-02-12 02-34-34](https://user-images.githubusercontent.com/26172160/74279574-26f49e80-4d41-11ea-936e-210f76f00c4a.png)

Reply:
@ac-optimus You're seeing this issue because you're running the Python interpreter in the directory containing `json.py`. When you're importing `json`, it's not importing from the standard library. It's importing from `json.py`.

If you run `cd $HOME; python -c "from json import load, loads"` you'll notice it works fine. This isn't an issue you'll see when importing `signac.core.json` in the package.

Reply:
@bdice  the first error comes when I add `from json import load, loads` in json.py while removing load and loads function as was suggested by you. I completely understand the point related to `cd $HOME; python -c "from json import load, loads"`. I was mentioning that while replacing `load` and `loads` methods from `json.py` with `from json import load, loads` I was welcomed by the error as in first snippet(If I am not committing some silly mistake then this can not be done). Also could you explain why am I getting this recursion when I just try to call load and loads methds from package's `json.py` when no change is made(In the both the snippets I am trying to import `json` from the package and not the standard library one?

Reply:
@ac-optimus I still believe the issue is caused by trying to run your Python interpreter in that directory. With the changes I just pushed, I get this output:

```python
~/code/signac/signac/core $ python
Python 3.7.6 | packaged by conda-forge | (default, Dec 27 2019, 00:09:34)
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bdice/code/signac/signac/core/json.py", line 5, in <module>
    from json import load, loads, JSONEncoder
ImportError: cannot import name 'load' from 'json' (/home/bdice/code/signac/signac/core/json.py)
>>>
```
This is expected, since I'm using `core/json.py` as a standalone module from the `core/` directory instead of as it is intended in the package.